code/game/machinery/constructable_frame.dm

```diff
@@ -1078,3 +1078,27 @@ to destroy them and players will be able to make replacements.
 	icon_state = "generic"
 	build_path = /obj/machinery/economy/merch
 	board_type = "machine"
+
+/obj/item/circuitboard/suit_storage_unit
+	board_name = "Suit Storage Unit"
+	icon_state = "generic"
+	build_path = /obj/machinery/suit_storage_unit
+	board_type = "machine"
+	origin_tech = "materials=2;engineering=2"
+	req_components = list(
+							/obj/item/stock_parts/matter_bin = 1,
+							/obj/item/stock_parts/micro_laser = 1,
+							/obj/item/stack/cable_coil = 3,
+							/obj/item/stack/sheet/rglass = 5)
+
+/obj/item/circuitboard/suit_storage_unit/industrial
```

Comment:
Why make it inherit suit_storage_unit board?

Reply:
The only difference between the two is the icon - one has a sliding metal door, the other is a glass.